prov/hook/trace/src/hook_trace.c

```diff
@@ -150,6 +150,34 @@ trace_cq(struct hook_cq *cq, const char *func, int line,
 	}
 }
 
+static inline void
+trace_cq_err(struct hook_cq *cq, const char *func, int line,
+             int count, struct fi_cq_err_entry *entry,  uint64_t flags)
+{
+	if ((count > 0) &&
+	    fi_log_enabled(cq->domain->fabric->hprov, FI_LOG_TRACE, FI_LOG_CQ)) {
```

Comment:
For fi_log_enable(), if it is false, just return from the function.  That way the rest of the function doesn't have an extra indentation.